main/process-tx.zkasm

```diff
@@ -60,7 +61,19 @@ checkAndSaveFrom:
         SR                              :MSTORE(initSR)
 
 ;;;;;;;;;;;;;;;;;;
-;; B - Verify chainID
+;; B - Verify tx.sender does not have any code deployed (EIP3607)
+;;;;;;;;;;;;;;;;;;
+
+        %SMT_KEY_SC_CODE => B
```

Comment:
We store in the state-tree the [length](https://github.com/0xPolygonHermez/zkevm-rom/blob/main/main/constants.zkasm#L16) of the contract (bytes) deployed, if it exist.
I would suggest to read that value from the state-tree. If 0, no contract has been deployed.
So, `codeLenght - 1   :JMPN(invalidIntrinsicTx)`

Since `codeLength` is set by the rom when a contract is deployed, we can assume that the value could not exceed 2³²
bytes. Therefore, we can use `JMPN` instead of a binary state-machine.
Also, given [this PR](https://github.com/0xPolygonHermez/zkevm-rom/pull/141) I would set a different name for the intrinsic error triggered, just to differentiate from the others